modules/user_accounts/templates/menu_user_accounts.tpl

```diff
@@ -2,7 +2,7 @@
 
 <!-- start the selection table -->
 <div class="row">
-    <div class="col-sm-9 col-lg-8">
+    <div class="col-sm-9 col-lg-12">
```

Comment:
Check with others about the look and feel of this. I'm not the biggest fan of the way it looks. I feel like we could use the white space in a better way. @taracampbell can you have a look at this. If people seem good with the way it looks then change this line to be `class="col-xs-12"`